src/pages/dashboard/Explore.tsx

```diff
@@ -150,11 +159,12 @@ const Explore = () => {
         url += queryObject.licenses.map((licenseCode) => `&${SearchParameters.license}=${licenseCode}`).join('');
       if (queryObject.tags.length > 0) url += queryObject.tags.map((tag) => `&${SearchParameters.tag}=${tag}`).join('');
       if (queryObject.showInaccessible) url += `&${SearchParameters.showInaccessible}=true`;
+      if (queryObject.offset > 0) url += `&${SearchParameters.page}=${queryObject.offset / 10 + 1}`;
```

Comment:
denne kan kanskje brukes her, for å slippe å bygge streng selv: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append

Reply:
Fikset